README.md

```diff
@@ -1,16 +1,16 @@
-<img width="150" height="150" align="left" style="float: left; margin: 0 10px 0 0;" alt="MS-DOS logo" src="https://github.com/Microsoft/MS-DOS/blob/master/msdos-logo.png">   
+<img width="150" height="150" align="left" style="float: left; margin: 0 10px 0 0;" alt="Logotipo do MS-DOS" src="https://github.com/Microsoft/MS-DOS/blob/master/msdos-logo.png">   
 
-# MS-DOS v1.25 and v2.0 Source Code
-This repo contains the original source-code and compiled binaries for MS-DOS v1.25 and MS-DOS v2.0.
+# Código Fonte do MS-DOS v1.25 e v2.0
+Este repositório contém o código fonte e os binários compilados originais do MS-DOS versão 1.25 e versão 2.0.
 
-These are the same files [originally shared at the Computer History Museum on March 25th, 2014]( http://www.computerhistory.org/atchm/microsoft-ms-dos-early-source-code/) and are being (re)published in this repo to make them easier to find, reference-to in external writing and works, and to allow exploration and experimentation for those interested in early PC Operating Systems.  
+Estes são os mesmos ficheiros [partilhados originalmente no Museu da História da Computação (Computer History Museum) no dia 25 de Março de 2014](http://www.computerhistory.org/atchm/microsoft-ms-dos-early-source-code/) e estão a ser (re)publicados neste repositório de forma a serem mais fáceis de se encontrar, referir em trabalhos ou artigos externos e/ou permitir a exploração/experimentação para aqueles interessados em Sistemas Operativos antigos destinados a PCs.
 
-# License
-All files within this repo are released under the [MIT (OSI) License]( https://en.wikipedia.org/wiki/MIT_License) as per the [LICENSE file](https://github.com/Microsoft/MS-DOS/blob/master/LICENSE.md) stored in the root of this repo.
+# Licença
+Todos os ficheiros deste repositório estão lançados sob a [Licença MIT (OSI)](https://pt.wikipedia.org/wiki/Licen%C3%A7a_MIT) como descrito no [ficheiro LICENSE](https://github.com/Microsoft/MS-DOS/blob/master/LICENSE.md) presente na raiz deste repositório.
 
-# Contribute!
-The source files in this repo are for historical reference and will be kept static, so please don’t send Pull Requests suggesting any modifications to the source files, but feel free to fork this repo and experiment 😊.  
+# Contribua!
+Os ficheiros de código fonte deste repositório são para referência histórica, logo não serão alterados. Deste modo, pedimos que não envie Pull Requests a sugerir quaisquer tipo de alterações a estes ficheiros, mas sinta-se à vontade de fazer um fork deste repositório e fazer alterações 😊.
```

Comment:
I'd use "não hesite" instead of "sinta-se à vontade" since it's a more natural way of putting it in PT-PT